### PR TITLE
Update lambda.md

### PR DESCRIPTION
**Pull request by JV0812**
## Purpose of this pull request

This pull request is to fix the FER scope (specific data) as [requested](https://sumologic.slack.com/archives/GFJHGKTC2/p1710166711465089).

## Select the type of change

- [x] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/integrations/amazon-aws/lambda.md [hunk 1]
```diff
@@ -286,16 +286,14 @@ Enter a parse expression to create an “account” field that maps to the alias
 ```yml
 Rule Name: AwsObservabilityLambdaCloudWatchLogsFER
 Applied at: Ingest Time
-Scope (Specific Data): _sourceHost=/aws/lambda/*
+Scope (Specific Data): account=*region* _sourceHost=/aws/lambda/*
```
Comment (npande) [inline docs/integrations/amazon-aws/lambda.md:289]: put a space before region
it should be account=* region=* _sourceHost=/aws/lambda/*

Comment (kimsauce) [inline docs/integrations/amazon-aws/lambda.md:289]:
```suggestion
Scope (Specific Data): account=* region* _sourceHost=/aws/lambda/*
```

Comment (kimsauce) [reply docs/integrations/amazon-aws/lambda.md]: done!

**Review by kimsauce — APPROVED** [review]

**Comment by himsharma01** [comment]
lgtm

**Pull request merged**